frontend/lib/parse-capi/index.ts

```diff
@@ -287,20 +287,30 @@ export const extractArticleMeta = (data: {}): CAPIType => {
         tag => tag.type === 'Contributor',
     )[0];
 
+    // From the server we get the values: "UK edition", "US edition", "Australia edition", "International edition"
+    // editionLongForm is that value, or empty string.
+    const editionLongForm = getString(data, 'config.page.edition', '');
+
+    // We compute the editionId from the editionLongForm
+    // Possible values for the editionId: "uk", "us", "au", "int"
```

Comment:
You can get this from the [id](https://github.com/guardian/frontend/blob/db014fcbfdc05549a38c8ae71b6c692e59e1a2c2/common/app/common/Edition.scala#L10) field in Edition.

Reply:
True. I have to make a change to frontend and then I will simplify this :)